docs/rst/notes/forthcoming_version.rst

```diff
@@ -27,3 +27,7 @@ Next release will include the following **major changes**:
 
 * ``wan`` Participant Kind uses now Initial Peers Discovery Protocol, while Discovery Server
   requires a new Participant Kind ``wan-discovery-server``.
+
+Next release will include the following **performance improvements**:
+
+* No locking in the :code:`Track` hot-path so transmission is executed faster.
```

Comment:
I would remove the last bit, as it has not been tested.

Reply:
I agree with this